[profiling] Dual-shipping notes. (#18424)
DataDog/documentation: content/en/agent/guide/dual-shipping.md

DD_APM_PROFILING_ADDITIONAL_ENDPOINTS='{\"https://intake.profile.datadoghq.com/api/v2/profile\": [\"apikey2\", \"apikey3\"], \"https://intake.profile.datadoghq.eu/api/v2/profile\": [\"apikey4\"]}'
```

> **Note:** Uploads to additional endpoints for the Continuous Profiler product are done through best-effort delivery.
> * The main endpoint has the highest priority. Uploads to additional endpoints are only handled after successful upload to the main endpoint.
> * Responses from additional endpoints are not forwarded back to the profiler. Any errors during delivery to additional endpoints are logged in the agent error logs.

Reviewer: ```suggestion
**Note:** Uploads to additional endpoints for the Continuous Profiler product are done through best-effort delivery.
* The main endpoint has the highest priority. Uploads to additional endpoints are only handled after uploads to the main endpoint have completed successfully.
* Responses from additional endpoints are not forwarded back to the profiler. Any errors during delivery to additional endpoints are logged in the Agent error logs.
```

> * Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints will be logged in the agent error logs.

Reviewer 2: ```suggestion
* Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints are logged in the agent error logs.
```

> * Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints will simply be logged in the agent error logs.

Reviewer 2: ```suggestion
* Responses from additional endpoints are not forwarded back to the continuous profiler. Any errors during delivery to additional endpoints will be logged in the agent error logs.
```
